scripts/superdesk/editor2/editor.spec.js

```diff
@@ -138,42 +138,6 @@ describe('text editor', function() {
 
     }));
 
-    it('can save model value', inject(function(editor, $rootScope) {
```

Comment:
Because the tested methods have been removed by this PR.
Now the history is kept at the global editor level, not anymore at the text block level